index.rst

```diff
@@ -74,15 +119,16 @@ The manual can be found on Docushare `here <https://docushare.lsst.org/docushare
 
 Fiber Spectrograph
 ------------------
-The fiber spectrograph used is an Avantes SensLine AvaSpecULS2048x64TEC-EVO, with a wavelength range of 200-1160 nm. An optical fiber runs from the fiber spectrograph and samples the light from the optical path after it exits from the monochromator. 
+The fiber spectrograph used is an Avantes SensLine AvaSpecULS2048x64TEC-EVO, with a wavelength range of 200-1160 nm. An optical fiber runs from the fiber spectrograph and samples the light from the optical path after it exits the monochromator. 
 
-The fiber spectrograph is controlled via USB that runs directly from the electrometer to a linux control computer (loonie) in the electronics cabinet. It can be commanded via CSC. 
+The fiber spectrograph is controlled via USB that runs directly from the fiber spectrograph to a linux control computer (loonie) in the electronics cabinet. It can be commanded via CSC. 
+
+We also have a fiber illuminator (Spectral Cal Lamp), which can be used to calibrate the Fiber Spectrograph. Power is avaialble for the illuminator on the table, but we expect that the illuminator will only be installed when needed. When we want to calibrate the fiber spectrograph, simply move the fiber from the optical path and install it into the illuminator.
 
-We also have a fiber illuminator, which can be used to calibrate the Fiber Spectrograph. Power is avaialble for the illuminator on the table, but we expect that the illuminator will only be installed when needed. When we want to calibrate the fiber spectrograph, simply move the fiber from the optical path and install it into the illuminator.
 
 Electrometer
 ------------
-The electrometer used is the Keithley 6517B. It reads the current from a Hammatsu Si S2281 Series photodiode. The photodiode samples the light on the edge of the optical path. It does not measure the total brightness but rather any changes in irradiance from day to day and even throughout a daily operation. 
+The electrometer used is the Keithley 6517B. It reads the current from a Hammatsu Si S2281 Series photodiode. The photodiode samples the light on the edge of the optical path after exiting the monochromator. It does not measure the total brightness but rather any changes in irradiance from day to day and even throughout a daily calibration. 
 
 The electrometer is controlled via a Serial Device Server, the MOXA Nport 5100. Reminder that this will need to be set to a baud rate of 9600. 
```

Comment:
this is wrong. The Baud is 56k, but there is more than that.
Information on the moxa setup is here:
https://ts-electrometer.lsst.io/developer-guide/developer-guide.html#moxa-serial-to-ethernet-converter
I suggest just linking this.

Reply:
included the link